File: app/src/main/java/kr/co/jayhy/weather/listener/MyWeatherCallback.java
package kr.co.jayhy.weather.listener;

import kr.co.jayhy.weather.repo.WeatherCurrentRepo;

/**
 * Created by jayhy on 2017. 8. 12..
 */

public interface MyWeatherCallback {

    void onMyWeatherComplate(boolean isSuccess, WeatherCurrentRepo result);
}

File: kotlin/src/main/java/kr/co/jayhy/weather/listener/MyWeatherCallback.kt
package kr.co.jayhy.weather.listener

import kr.co.jayhy.weather.repo.WeatherCurrentRepo

/**
 * Created by jayhy on 2017. 8. 12..
 */

interface MyWeatherCallback {

    fun onMyWeatherComplate(isSuccess: Boolean, result: WeatherCurrentRepo)
}
